sdk/src/main/java/com/mapbox/maps/MapView.kt

```diff
@@ -36,7 +36,7 @@ import com.mapbox.maps.viewannotation.ViewAnnotationManagerImpl
  * and for ensuring your use adheres to the relevant terms of use.
  */
 open class MapView : FrameLayout, MapPluginProviderDelegate, MapControllable {
-  private var mapController: MapController
+  internal var mapController: MapController
```

Comment:
same question about private setter

Reply:
`val` could not have a setter.

Reply:
agreed about comment above, but here `var`

Reply:
fixed.